src/main/resources/data/invoice-lines/82912-1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "33dc5f4c-0387-4359-919b-9f654c757b54",
+  "adjustments": [
+    {
+      "description":"Shipping",
+      "type":"Amount",
+      "value":10.00,
```

Comment:
The PO Lin has discount 20 USD i.e. negative adjustment is expected

Reply:
Done